src/types.rs

```diff
@@ -81,6 +81,7 @@ pub enum BufferType {
     GrpcReceiveBuffer = 5,
     VmConfiguration = 6,
     PluginConfiguration = 7,
+    ForeignFunctionArguments = 8,
```

Comment:
This should be named `CallData` to align with Proxy-Wasm C++ SDK, like for the rest of the type names.